pkgs/development/mobile/genymotion/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   name = "genymotion-${version}";
   version = "2.8.0";
   src = requireFile {
-    url = https://www.genymotion.com/download/;
+    url = https://dl.genymotion.com/releases/genymotion-2.8.0/genymotion-2.8.0-linux_x64.bin;
```

Comment:
https://dl.genymotion.com/releases/genymotion-3.0.2/genymotion-3.0.2-linux_x64.bin looks valid too so you could replace the 2.8.0 with ${version}